source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,517 @@
+  available version. If the instance falls more than one major version 
+  behind the latest available, it enters a :guilabel:`Failed` state and cannot
+  sync with Atlas until it is upgraded. For more information, refer to 
+  :ref:`upgrade-edge-server-version`.
+- View the Install command that you can use in the terminal to install the 
+  Edge Server code.
+- Regenerate the registration token.
+- Delete the Edge Server instance.
+
+.. _delete-edge-server-instance:
+
+Delete an Edge Server Instance
+------------------------------
+
+You can delete an individual Edge Server instance from Atlas.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Browse to the Edge Server Dashboard
+
+            Click the :guilabel:`Edge Server` option in the left navigation 
+            sidebar to open the Edge Server dashboard.
+
+         .. step:: View the Edge Server Instance Details
+
+            Find the Edge Server instance you want to delete in the list,
+            and click the :guilabel:`View Details` button.
+
+         .. step:: Delete the Edge Server Instance
+
+            Click the :guilabel:`Delete` button at the bottom of the Edge
+            Server details view.
+
+            You'll be prompted to confirm deletion. Follow the prompts to
+            confirm that you want to delete the Edge Server.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Get the ID of the Edge Server to Delete
+
+            Call the <LIST EDGE SERVER> endpoint to get a list of all the 
+            Edge Server instances:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            Find the ID of the Edge Server instance that you want to delete 
+            in the list.
+
+         .. step:: Call the Delete Endpoint
+
+            Call the <DELETE EDGE SERVER> endpoint with the ID of the Edge
+            Server instance that you want to delete.
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+.. _update-edge-server-data-model:
+
+Update an Edge Server's Data Model
+----------------------------------
+
+The process to update a data model varies depending on whether you're using
+an Edge Server Service or a Device Sync Service with Edge Server.
+
+Edge Server Service
+~~~~~~~~~~~~~~~~~~~
+
+When you're using an Edge Server Service, the service automatically updates
+fields in the schema when you make changes to the documents you sync.
+
+If you update your schema manually using the UI, and those schema changes
+trigger a schema version change, this causes an Edge Server instance with the 
+old schema to reboot and wipe itself.
+
+If you're adding a new collection you haven't synced before, you must define
+a new schema.
+
+Add a Collection
```

Comment:
I think we want to add here that the Edge Server needs to be rebooted to pick up on new collections.

Reply:
Noted this in ln 572 below - thanks for calling this out!